### ISSUE TITLE
Stacked column legend toggle does not work on builds beyond 1.1.2

### ISSUE THREAD

**Issue by bloodhawk** [n3-charts/line-chart#191]
The example at: bit.ly/1D0EAEN does not work on builds above version 1.1.2. If I use a higher build I end up getting an error: TypeError: Cannot read property 'y' of undefined

```
TypeError: Cannot read property 'y' of undefined
    at http://localhost:3000/js/lib/line-chart.js:1258:37
```

From what I was able to find is that in the function getVerticalDomain it calls a filter as so:

```
domain = this.yExtent(series.filter(function(s) {
          return s.axis === key && s.visible !== false;
        })
```
The logic ``` && s.visible !== false ``` seems to be stripping off the series that is not currently visible. This later causes an issue because it tries to read off a property of undefined in the function yExtent.

```
 return maxY = Math.max(maxY, d3.max(data, function(d) {
            return group.reduce((function(a, s) {
              return a + d[s.y]; 
            }), 0);
          }));
```
is where the error is thrown. I ended up removing that logic but it did not solve the issue completely. When removing that logic, if all keys are toggled off in the legend the y axis disappears completely and breaks the graph. From what I can tell this is only happening with stacked graphs like the one shown at bit.ly/1D0EAEN.

Here is a fiddle replicating the issue:
http://jsfiddle.net/bloodhawk/ozx38asw/

**Comment by lorem--ipsum** [comment]
See PR. Thanks :-)

**Comment by bloodhawk** [comment]
Thanks for getting back to me. Does this mean an alternative fix will be implemented at some point for this issue. Legend toggles cease to work with stacked bar graphs. Do I need to wait on https://github.com/n3-charts/line-chart/issues/146?

**Comment by lorem--ipsum** [comment]
Hm. I've closed that issue a little bit too fast. I'll take a look at this legend toggle problem.

**Comment by lorem--ipsum** [comment]
Allright, can you try with `master` please ?

**Comment by Shipow** [comment]
reproduced with master (and also not working on the examples page)

**Comment by lorem--ipsum** [comment]
Okay, this is fixed in current (1.1.8).
